airbyte-integrations/connectors/source-gitlab/acceptance-test-config.yml

```diff
@@ -18,7 +18,7 @@ acceptance_tests:
     tests:
       - config_path: "secrets/config.json"
         backward_compatibility_tests_config:
-          disable_for_version: 1.7.1
+          disable_for_version: 1.8.4
```

Comment:
if we're not introducing a breaking change, this entry may be removed

Reply:
This was added because of adding/changing format for date/date-time string fields, which is non-breaking change.